src/app/item-popup/ItemStat.tsx

```diff
@@ -122,58 +119,12 @@ export default function ItemStat({ stat, item }: { stat: DimStat; item?: DimItem
         </div>
       )}
 
-      {stat.bar && (
-        <div
-          className={styles.statBar}
-          aria-label={stat.displayProperties.name}
-          title={stat.displayProperties.description}
-          aria-hidden="true"
-        >
-          <div className={styles.barContainer}>
-            {segments.map(([val, className], index) => (
-              <div
-                key={index}
-                className={clsx(styles.barInner, className)}
-                style={{ width: percent(val / stat.maximumValue) }}
-              />
-            ))}
-          </div>
-        </div>
-      )}
+      {stat.bar && <StatBar {...{ stat, segments }} />}
```

Comment:
I generally don't like spreading props into components, mostly because it makes it easy to pass more properties than the component actually needs (see the warning in https://reactjs.org/docs/jsx-in-depth.html#spread-attributes) though maybe TypeScript is saving our bacon here?